docs/GettingStarted.md

```diff
@@ -172,7 +120,9 @@ retrieved at the Rack level.
 To start using the middleware in your generic Rack application, add it to your ``config.ru``:
 
     # config.ru example
-    require 'ddtrace/contrib/rack/middlewares'
+    require 'ddtrace'
+
+    Datadog.configure { |c| c.use(:rack) }
```

Comment:
Minor: although `{ }` is equivalent to the block style syntax in the other examples, might it be good for consistency/readability to use `do ... end`?

Reply:
Agreed. I'll change that